.github/workflows/check_csv_data_integrity.yaml

```
@@ -0,0 +1,117 @@
+                echo "❌ ERROR: CSV length changed for $file (was $BASE_LINES, now $CURRENT_LINES)"
+                exit 1
+              fi
+            fi
+          done
+          
+          echo "✅ All CSV files maintain their original length"
+
+      - name: Check for data deletion
+        run: |
+          echo "🔍 Checking for data deletion in poster_data/data..."
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)